CHANGELOG.md

```diff
@@ -31,6 +31,7 @@ To make use of filtering, configure `autocomplete_filtering_enabled`.
 * [ENHANCEMENT] Add support for IPv6 [#1555](https://github.com/grafana/tempo/pull/1555) (@zalegrala)
 * [ENHANCEMENT] Add span filtering to spanmetrics processor [#2274](https://github.com/grafana/tempo/pull/2274) (@zalegrala)
 * [BUGFIX] tempodb integer divide by zero error [#2167](https://github.com/grafana/tempo/issues/2167) (@kroksys)
+* [BUGFIX] metrics-generator: use fanout storage to correctly update Prometheus remote write [#2463](https://github.com/grafana/tempo/issues/2463) (@kvrhdn)
```

Comment:
Will an end-user understand this?
Suggest to focus more on the effect, which I think is Tempo will now use multiple connections to send samples as the rate gets higher.

Reply:
Good point thanks! I tried to improve it somewhat